src/main/resources/lang/en-US.yml

```diff
@@ -335,7 +335,7 @@ msg_town_immunity_expired: 'Your town''s Siege Immunity has expired.'
 
 #Unsiegeable towns strings
 msg_permanent: "permanent"
-msg_error_must_be_num_or_permanent: '&cAmount must be a number or ''permanent''.'
+msg_error_must_be_num_or_permanent: "&cAmount must be a number or ''permanent''."

```

Comment:
This change is not necessary, the '' '' surrounding permanent allows the string to be wrapped in '' without issue. 

If this does get changed to being wrapped in ""'s then the permanent can change to `'permanent'` but it will also mean that every language on crowdin will have their version of this string changed back to the new English version, adding work on the translators.